localization/ekf_localizer/include/ekf_localizer/matrix_types.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018-2019 Autoware Foundation
```

Comment:
@IshitaTakeshi Please confirm that the copyright is correct.

Reply:
Updated in 89814787cb67ea6de062e74358e6263c995a2099

Reply:
@mitsudome-r @kenji-miyake 

How can I write the copyright notice? I copied the existing one and modified the year like below.

```
// Copyright 2022 Autoware Foundation
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

But the [license page](https://autowarefoundation.github.io/autoware-documentation/main/contributing/license/) shows the example below.

```
Copyright [first year of contribution] The Autoware Contributors
SPDX-License-Identifier: Apache-2.0
```

Which one should I use?

Reply:
> Currently, the old formats explained [here](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/blob/87c5e5880a18068116dd886ad56e1bfc29e694c4/CONTRIBUTING.md) are also acceptable. Those old formats can be replaced by this new format if the original authors agree.

Reply:
@YamatoAndo Can I merge this?